apps/ae_socket_connector/lib/client_runner.ex

```diff
@@ -20,7 +21,10 @@ defmodule ClientRunner do
   end
 
   defp log_callback(type, round, round_initiator, method, color) do
-    Logger.debug("received: #{inspect(type)}, #{inspect(round)}, #{inspect(round_initiator)}, #{inspect(method)}", color)
+    Logger.debug(
+      "received: #{inspect(type)}, #{inspect(round)}, #{inspect(round_initiator)}, #{inspect(method)}",
```

Comment:
In such cases you can also simply inspect a tuple with these elements, e.g. `#{inspect({type, round, round_initiator, method})}`

Reply:
fixed